tools/integration_tests/util/setup/setup.go

```diff
@@ -460,3 +460,22 @@ func RunTestsOnlyForStaticMount(mountDir string, t *testing.T) {
 		t.SkipNow()
 	}
 }
+
+// AppendFlagsToAllFlagsInTheFlagsSet appends the newFlags in every flags present in the
+// flagsSet.
+// Input flagsSet: [][]string{{"--x", "--y"}, {"--x", "--z"}}
+// Input newFlags: "--a", "--b"
+// Output modified flagsSet: [][]string{{"--x", "--y", "--a", "--b"}, {"--x", "--z", "--a", "--b"}}
```

Comment:
Shouldn't it be
`Output modified flagsSet: [][]string{{"--x", "--y", "--a"}, {"--x", "--z", "--a"},{"--x", "--y", "--b"},{"--x", "--z", "--b"}}`

Also, I think a better example would be 

// Input flagsSet: [][]string{{"--x", "--y"}, {"--x", "--z"}}
// Input newFlags: {"--a", "--b", ""}
// Output modified flagsSet: [][]string{{"--x", "--y", "--a"}, {"--x", "--z", "--a"},{"--x", "--y", "--b"},{"--x", "--z", "--b"},{"--x", "--y"}, {"--x", "--z"}}